config/routes.rb

```diff
@@ -7,6 +7,7 @@
 
   resources :cart_items, only: [:create, :destroy, :update]
   resources :items, only: [:index, :show]
+  get '/deal', to: 'items#deal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.